pkg/cri/server/sandbox_run.go

```diff
@@ -96,7 +96,8 @@ func (c *criService) RunPodSandbox(ctx context.Context, r *runtime.RunPodSandbox
 			RuntimeHandler: r.GetRuntimeHandler(),
 		},
 		sandboxstore.Status{
-			State: sandboxstore.StateUnknown,
+			State:     sandboxstore.StateUnknown,
+			CreatedAt: time.Now().UTC(),
```

Comment:
The same change will have to be made in the `sbserver` package also 

https://github.com/containerd/containerd/blob/release/1.7/pkg/cri/sbserver/sandbox_run.go#L122

Reply:
silly bot doesn't understand though 😆

Reply:
- https://github.com/containerd/containerd/pull/10122